Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/workspaceOptions/CopyAndPasteController.java

```diff
@@ -277,7 +277,7 @@ public void cut(Set<Object> selections, int selectionCount, FolderType currentFo
     if (checkContext(selections, selectionCount, currentFolder)) {
       try {
         if (!checkImmediateOwnership(selections)) {
-          Notification.error(Labels.getLabel("portal_onlyOwnerCanCutItems"));
+          Notification.error(Labels.getLabel("portal_onlyOwnerCanCutItems_message"));
```

Comment:
This is different from the original PR, but it's correct if you check against Apromore-Core-Components/Apromore-Portal/src/main/resources/zk-label.properties